dev/tasks/r/github.packages.yml

```diff
@@ -84,6 +135,8 @@ jobs:
         env:
           UBUNTU: {{ '"${{ matrix.ubuntu }}"' }}
         {{ macros.github_set_sccache_envvars()|indent(8) }}
+          MACOSX_DEPLOYMENT_TARGET: "10.13"

```

Comment:
Is this needed here under `linux-cpp` / `C++ Binary Linux OpenSSL {{ '${{ matrix.openssl }}' }}`?